addons/common/init_perFrameHandler.sqf

```diff
@@ -80,6 +80,22 @@ GVAR(waitUntilAndExecArray) = [];
     if (_delete) then {
         GVAR(waitUntilAndExecArray) = GVAR(waitUntilAndExecArray) - [objNull];
     };
+
+    // Execute the waitUntilAndExecTimeout functions:
+    _delete = false;
```

Comment:
could move this line after L81, so the flag is only reset if it ever where set. I think the same can be done above that once too.